mvn-install

```diff
@@ -19,23 +19,23 @@ CASSANDRA_BUNDLE_ID=$(conduct load cassandra --long-ids -q)
 conduct run ${CASSANDRA_BUNDLE_ID} --no-wait -q
 
 echo "Deploying friend-impl..."
-FRIEND_IMPL_BUNDLE_ID="$(docker save chirper/friend-impl | bndl --no-default-check --endpoint friendservice --bind-protocol http --bind-port 0 --service-name friendservice --acl http:/api/users --endpoint akka-remote | conduct load --long-ids -q)"
+FRIEND_IMPL_BUNDLE_ID="$(docker save chirper/friend-impl | bndl --no-default-check --endpoint friendservice --bind-protocol http --bind-port 0 --service-name friendservice --acl /api/users --path-beg --endpoint akka-remote | conduct load --long-ids -q)"
 conduct run "$FRIEND_IMPL_BUNDLE_ID" --no-wait -q
 
 echo "Deploying load-test-impl"
 LOAD_TEST_IMPL_BUNDLE_ID="$(docker save chirper/load-test-impl | bndl --no-default-check --endpoint loadtestservice --bind-protocol http --bind-port 0 --service-name loadtestservice --endpoint akka-remote | conduct load --long-ids -q)"
 conduct run "$LOAD_TEST_IMPL_BUNDLE_ID" --no-wait -q
 
 echo "Deploying chirp-impl..."
-CHIRP_IMPL_BUNDLE_ID="$(docker save chirper/chirp-impl | bndl --no-default-check --endpoint chirpservice --bind-protocol http --bind-port 0 --service-name chirpservice --acl http:/api/chirps/live --acl http:/api/chirps/history --endpoint akka-remote | conduct load --long-ids -q)"
+CHIRP_IMPL_BUNDLE_ID="$(docker save chirper/chirp-impl | bndl --no-default-check --endpoint chirpservice --bind-protocol http --bind-port 0 --service-name chirpservice --acl /api/chirps/live --path-beg --acl /api/chirps/history --path-beg --endpoint akka-remote | conduct load --long-ids -q)"
```

Comment:
I think `--acl /api/chirps/history --path-beg` can be exposed without path begin match: `--acl /api/chirps/history` since `ChirpService` has exact URL `/api/chirps/history`:

https://github.com/lagom/activator-lagom-java-chirper/blob/master/chirp-api/src/main/java/sample/chirper/chirp/api/ChirpService.java#L28